terraform/README.md

```diff
@@ -24,9 +24,11 @@ IBM Cloud Schematics delivers Terraform-as-a-Service so that you can use a high-
 
 Follow these two easy steps to create an IBM Cloud Schematics workspace and apply a Terraform template. When you apply the template, a Kubernetes cluster is created, and a Node.js Express application is deployed to it by using an IBM Cloud DevOps toolchain.
 
+There are two types of IBM Cloud DevOps toolchain options available with this application: classic and GitSecure. The classic toolchain has a code repository, delivery pipeline, and web IDE. The GitSecure toolchain has the same features as the classic, but also includes the Vulnerability Advisor to help developers design a secure app with little effort.
```

Comment:
Replace this string:
"There are two types of IBM Cloud DevOps toolchain options available with this application:"

with this string:
"Two types of IBM Cloud DevOps toolchain options are available with this application:"